velox/docs/functions/spark/json.rst

```diff
@@ -0,0 +1,13 @@
+==============
```

Comment:
Would you add this documentation in https://github.com/facebookincubator/velox/blob/main/velox/docs/spark_functions.rst?plain=1#L8-L22 to let it be displayed in spark functions?